integration_tests/graalvm_tests/pom.xml

```diff
@@ -59,6 +58,22 @@
       <artifactId>fory-core</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.fory</groupId>
+      <artifactId>fory-graalvm-feature</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.graalvm.sdk</groupId>
```

Comment:
`fory-graalvm-feature` should alreay has `org.graalvm.sdk` as provided , why add it again here?